Documentation/applications/testing/open_memstream/index.rst

```diff
@@ -0,0 +1,11 @@
+=============================================
+``open_memstream`` - open_memstream test tool
+=============================================
+
+``CONFIG_TESTING_OPEN_MEMSTREAM_TEST=y``
```

Comment:
should we remove _TEST suffix?

Reply:
I think we can. It seems we use a bit of everything in `apps/testing` though (both without, with `_TEST` and with `TEST` but without `_`).

Reply:
_TEST suffix with TESTING_ prefix looks redundant.